setup.py

```diff
@@ -324,29 +324,24 @@ def run_tests(self):
         sys.exit(main())
 
 
-class BinaryDistApp(PyInstallerDist):
-    description = 'create an application for OSX'
-    extra_args = [
-        '--icon=osx/plover.icns',
-        '--osx-bundle-identifier=org.openstenoproject.plover',
-    ]
+class BinaryDistApp(Command):
+    description = 'create an application bundle for Mac'
+    user_options = []
+    extra_args = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
 
     def run(self):
-        PyInstallerDist.run(self)
-        # Patch Info.plist.
-        plist_info_fname = 'dist/%s.app/Contents/Info.plist' % PACKAGE
-        tree = ElementTree.parse(plist_info_fname)
-        dictionary = tree.getroot().find('dict')
-        for name, value in (
-            # Enable retina display support.
-            ('key'   , 'NSPrincipalClass'),
-            ('string', 'NSApplication'   ),
-        ):
-            element = Element(name)
-            element.text = value
-            element.tail = '\n'
-            dictionary.append(element)
-        tree.write(plist_info_fname)
+        self.run_command('bdist_wheel')
+        whl_cmd = self.get_finalized_command('bdist_wheel')
```

Comment:
Why not use `whl_cmd.run()` so you don't have to call `self.run_command('bdist_wheel')`?